messages/run.js

```diff
@@ -58,13 +60,17 @@ module.exports = {
 	Specify tsconfig.json if the current working directory does not contain the tsconfig.json that corresponds to the TypeScript files being scanned.
 		E.g., sfdx scanner:run --target "/my-project/**/*.ts" --tsconfig "/my-project/tsconfig.json"
 			Scans the project contained in '/my-project' if the current working directory is another directory.
-			
+
 	Use --env to override the default ESLint environment variables to add frameworks.
 		E.g., $ sfdx scanner:run --target "somefile.js" --env '{"jasmine": true}'
 			Evaluates rules against somefile.js, including Jasmine in the environment variables.
-	
+
 	Use --violations-cause-error to throw exit with a non-zero code when violations are found.
 		E.g., $ sfdx scanner:run --target "somefile.js" --violations-cause-error
 			Evaluates rules against somefile.js. If any rules are violated, the exit code will be the severity of the most severe violation.
-`
+
+	Use --engines to include or exclude engines. Any engine listed will be run, regardless of its current 'disabled' attribute.
+		E.g., $ sfdx scanner:run --target "somefile.js" --engines "eslint-lwc"
```

Comment:
I think it's for the best if the example shows using multiple engines instead of just one i.e. `--engines "eslint-lwc,pmd"`.
Also, is it valid to include both `eslint-lwc` and `eslint`?
Also-also, this should be `--engine` instead of `--engines`, since the flag itself is singular.

Reply:
I will add pmd as another example. 

I had the same question about `engines` vs. `engine` and followed the existing pattern of `category` and `ruleset`. Did the CLI board approve of the first set of flags?